SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -460,6 +459,34 @@ void USpatialSender::SendComponentInterestForActor(USpatialActorChannel* Channel
 	NetDriver->Connection->SendComponentInterest(EntityId, CreateComponentInterestForActor(Channel, bNetOwned));
 }
 
+void USpatialSender::SendInterestComponentChange(const Worker_EntityId EntityId, const Worker_ComponentId OldComponent, const Worker_ComponentId NewComponent)
```

Comment:
nit: function name is misleading as it doesn't affect the interest component- this is the code for dynamic component add/remove which changes which NCD bucket the entity falls into right? Could this be named `SendNCDBucketComponentChange`?

Reply:
Yep fair point, will update

Reply:
We've paired the Always_relevant marker in here also, so I'll update to `SendInterestBucketComponentChange`.